app/react/Metadata/components/MetadataFormButtons.js

```diff
@@ -19,13 +19,15 @@ export class MetadataFormButtons extends Component {
     const data = this.props.data.toJS();
 
     const ViewButton = (
-      <I18NLink to={`entity/${data.sharedId}`}>
-        <button type="button" className="edit-metadata btn btn-primary">
```

Comment:
@RafaPolit @daneryl @konzz Any idea why this button was wrapped with an anchor `a` tag?

Reply:
no idea, this is not needed, the change u made should be good.